lib/kvdb/kvdb_home.c

```diff
@@ -115,7 +115,7 @@ kvdb_home_pidfile_path_get(const char *home, char *buf, const size_t buf_sz)
 
     int n;
 
-    n = snprintf(buf, buf_sz, "%s/" PIDFILE_NAME, home);
+    n = snprintf(buf, buf_sz, "%s%s" PIDFILE_NAME, home, home[strlen(home) - 1] == '/' ? "" : "/");
```

Comment:
If you really want to clean up dups, you might need to loop on last char and backup until you're not sitting at a `/`.

Reply:
But since we can't easily eliminate dups in the middle, maybe we shouldn't worry about dups anywhere in path.

Reply:
I have a different PR for removing dup slashes like that in the works. Plan is to handle it at the kvdb_interface level.

Reply:
Do we disallow relative paths in kvdb_home?

Reply:
No we don't, but absolute paths are stored or mclass paths don't sit within the home